libzeth/tests/circuits/note_test.cpp

```diff
@@ -10,19 +10,17 @@
 #include "libzeth/core/merkle_tree_field.hpp"
 #include "libzeth/core/note.hpp"
 #include "libzeth/core/utils.hpp"
+#include "zeth_config.h"
 
 #include <gtest/gtest.h>
 
 using namespace libzeth;
 
-using ppT = libzeth::ppT;
-
-// Should be alt_bn128 in the CMakeLists.txt
-using FieldT = libff::Fr<ppT>;
-
 // We use our hash functions to do the tests
-using HashT = BLAKE2s_256<FieldT>;
-using HashTreeT = MiMC_mp_gadget<FieldT>;
+using pp = defaults::pp;
+using FieldT = defaults::FieldT;
```

Comment:
Same question as above, re: `FieldT`->`Field`

Reply:
Yes, technically we should.  I was hesitant because of the number of lines it might touch, but since you raised it I'll include this change.

Reply:
I've switched this everywhere.  I think we should now be consistent with the type var vs type name convention.

Reply:
> Yes, technically we should. I was hesitant because of the number of lines it might touch, but since you raised it I'll include this change.

Thanks @dtebbs 
Yes, I think it's important to be consistent here